keyboards/tab/swagkeys/transition_lite_1k/keyboard.json

```diff
@@ -0,0 +1,300 @@
+            {"matrix": [2, 8], "x": 112, "y": 27, "flags": 4},
+            {"matrix": [2, 9], "x": 125, "y": 27, "flags": 4},
+            {"matrix": [2, 10], "x": 137, "y": 27, "flags": 4},
+            {"matrix": [2, 11], "x": 150, "y": 27, "flags": 4},
+            {"matrix": [2, 12], "x": 163, "y": 27, "flags": 4},
+            {"matrix": [2, 13], "x": 179, "y": 27, "flags": 4},
+            {"matrix": [3, 15], "x": 198, "y": 27, "flags": 4},
+            {"matrix": [2, 14], "x": 211, "y": 27, "flags": 4},
+            {"matrix": [2, 15], "x": 224, "y": 27, "flags": 4},
+            {"matrix": [3, 12], "x": 174, "y": 40, "flags": 4},
+            {"matrix": [3, 11], "x": 153, "y": 40, "flags": 4},
+            {"matrix": [3, 10], "x": 141, "y": 40, "flags": 4},
+            {"matrix": [3, 9], "x": 128, "y": 40, "flags": 4},
+            {"matrix": [3, 8], "x": 115, "y": 40, "flags": 4},
+            {"matrix": [3, 7], "x": 102, "y": 40, "flags": 4},
+            {"matrix": [3, 6], "x": 89, "y": 40, "flags": 4},
+            {"matrix": [3, 5], "x": 76, "y": 40, "flags": 4},
+            {"matrix": [3, 4], "x": 64, "y": 40, "flags": 4},
+            {"matrix": [3, 3], "x": 51, "y": 40, "flags": 4},
+            {"matrix": [3, 2], "x": 38, "y": 40, "flags": 4},
+            {"matrix": [3, 1], "x": 25, "y": 40, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 40, "flags": 4},
+            {"matrix": [4, 0], "x": 11, "y": 52, "flags": 4},
+            {"matrix": [4, 1], "x": 32, "y": 52, "flags": 4},
+            {"matrix": [4, 2], "x": 44, "y": 52, "flags": 4},
+            {"matrix": [4, 3], "x": 57, "y": 52, "flags": 4},
+            {"matrix": [4, 4], "x": 70, "y": 52, "flags": 4},
+            {"matrix": [4, 5], "x": 83, "y": 52, "flags": 4},
+            {"matrix": [4, 6], "x": 96, "y": 52, "flags": 4},
+            {"matrix": [4, 7], "x": 109, "y": 52, "flags": 4},
+            {"matrix": [4, 8], "x": 121, "y": 52, "flags": 4},
+            {"matrix": [4, 9], "x": 134, "y": 52, "flags": 4},
+            {"matrix": [4, 10], "x": 147, "y": 52, "flags": 4},
+            {"matrix": [4, 11], "x": 171, "y": 52, "flags": 4},
+            {"matrix": [4, 14], "x": 211, "y": 52, "flags": 4},
+            {"matrix": [5, 15], "x": 224, "y": 64, "flags": 4},
+            {"matrix": [5, 14], "x": 211, "y": 64, "flags": 4},
+            {"matrix": [3, 13], "x": 198, "y": 64, "flags": 4},
+            {"matrix": [5, 11], "x": 179, "y": 64, "flags": 4},
+            {"matrix": [5, 10], "x": 163, "y": 64, "flags": 4},
+            {"matrix": [5, 9], "x": 147, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 120, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 106, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 92, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 79, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 65, "y": 64, "flags": 4},
```

Comment:
The LED index to matrix position mapping must be 1:1. Multiple LEDs assigned to the same key will not work properly.